hrms/payroll/doctype/salary_structure/salary_structure.js

```diff
@@ -127,12 +127,13 @@ frappe.ui.form.on("Salary Structure", {
 
 	assign_to_employees:function (frm) {
 		var d = new frappe.ui.Dialog({
-			title: __("Assign to Employees"),
+			title: __("Bulk Salary Structure Assignment"),
 			fields: [
-				{fieldname: "sec_break", fieldtype: "Section Break", label: __("Filter Employees By (Optional)")},
-				{fieldname: "grade", fieldtype: "Link", options: "Employee Grade", label: __("Employee Grade")},
-				{fieldname:'department', fieldtype:'Link', options: 'Department', label: __('Department')},
+				{fieldname: "sec_break", fieldtype: "Section Break", label: __("Employee Filters")},
+				{fieldname: "branch", fieldtype: "Link", options: "Branch", label: __("Branch")},
```

Comment:
You will have to wire the branch filter on the server side

<img width="1440" alt="image" src="https://github.com/frappe/hrms/assets/24353136/71d653a9-d2b5-44c1-8ef6-4c4f042e9b3e">

Reply:
My bad 💀